src/runtime/auto-prompt-manager-test.js

```diff
@@ -1164,59 +1136,67 @@ describes.realWin('AutoPromptManager', (env) => {
     await autoPromptManager.showAutoPrompt({
       autoPromptType: AutoPromptType.CONTRIBUTION,
       alwaysShow: true,
-      displayLargePromptFn: alternatePromptSpy,
     });
     expect(logEventSpy).to.be.calledOnceWith(expectedEvent);
-    expect(alternatePromptSpy).to.be.calledOnce;
   });
 
-  it('should replace the contribution miniprompt with a large prompt if DISABLE_DESKTOP_MINIPROMPT is enabled and viewport is wider than 480px', async () => {
-    win./*OK*/ innerWidth = 500;
-    setExperiment(win, ExperimentFlags.DISABLE_DESKTOP_MINIPROMPT, true);
-    miniPromptApiMock.expects('create').never();
-
-    await autoPromptManager.showAutoPrompt({
-      autoPromptType: AutoPromptType.CONTRIBUTION,
-      alwaysShow: true,
-      displayLargePromptFn: alternatePromptSpy,
-    });
-    expect(alternatePromptSpy).to.be.calledOnce;
-  });
-
-  it('should replace the subscription miniprompt with a large prompt if DISABLE_DESKTOP_MINIPROMPT is enabled and viewport is wider than 480px', async () => {
-    win./*OK*/ innerWidth = 500;
-    setExperiment(win, ExperimentFlags.DISABLE_DESKTOP_MINIPROMPT, true);
-    miniPromptApiMock.expects('create').never();
+  it(
+    'should replace the contribution miniprompt with a large prompt if ' +
```

Comment:
Could we avoid splitting the names up like this? I noticed it leads to more indentation for the body of the tests
![image](https://github.com/subscriptions-project/swg-js/assets/1216762/19490906-b972-426d-b7c2-5cf9b1201fba)

Reply:
I had thought the line length was part of the style guide but I guess that isn't the case. Reverting these changes so the formatting is unchanged.